### DISCUSSION TITLE
官方交流渠道：QQ 群

### DISCUSSION THREAD

**Discussion by Yidadaa** [ChatGPTNextWeb/NextChat#1724] [Announcements]
为了提高交流效率，我们设立了官方 QQ 群，如果你在使用或者搭建过程中遇到了任何问题，请先第一时间加群或者频道咨询解决，除非是可以稳定复现的 Bug 或者较为有创意的功能建议，否则请不要随意往 Issue 区发送低质无意义帖子。

点击下方蓝色字体加入对应群聊，二维码会定时更新。

[QQ 群](https://github.com/user-attachments/assets/d78421e3-9017-4ba4-8a14-576c10879a3b)

如果二维码过期，请尝试搜索 QQ 群：`746068031` 。

**Comment by delbug** [comment]
You exceeded your current quota, please check your plan and billing details. 纳尼？

**Comment by buzaimimang** [comment]
群满了  能开个新群吗

**Comment by kaynai** [comment]
请问作者大大百度统计代码应该在哪里添加

**Comment by DaLuY** [comment]
能再开个群么

**Reply by RainFung** [reply]
群满

**Comment by Yidadaa** [comment]
8 群: 370639856

**Reply by allen2fuc** [reply]
大哥这个群已经满了

**Reply by sxchou** [reply]
加7群705100933

**Reply by syctech** [reply]
有谁遇到这个问题的吗？api的配额没有超，但是总是提示“You exceeded your current quota, please check your plan and billing details.”

**Reply by PSBgerm** [reply]
↑我也是,一直提醒余额,但是根本没超

**Reply by shijie-dev** [reply]
麻烦也拉我一下.谢谢了1643647730

**Comment by gitHubDingGuo** [comment]
群满了。

**Reply by Dmw9527** [reply]
没有新的群吗，都满了诶

**Reply by Yidadaa** [reply]
8 群人数已扩展到 500 人。

**Reply by ffreemt** [reply]
桌面qq里搜 chatgptn4web 好像无结果。不过已进8群，感谢。

**Reply by Poker168** [reply]
都满了

**Reply by EndDream** [reply]
群已经满了

**Comment by linglom1** [comment]
大佬还考虑创建一个新群吗

**Reply by lujian9527** [reply]
@Yidadaa 建群咯，群主

**Comment by Yidadaa** [comment]
9 群：754611012

**Reply by zlpu** [reply]
大佬，建群了，有问题想咨询

**Reply by lujian9527** [reply]
群主，看见了请拉我一下673131865，9群也满了，加不进了咯

**Reply by chihiro365yb** [reply]
麻烦也拉下我，2717857882

**Comment by ghost** [comment]
請問授權碼怎麼獲得啊？

**Comment by shenxianbu4** [comment]
我使用docker创建成功后，ubuntu系统在哪里查看与chatgpt的问答的日志文件？

**Reply by shenxianbu4** [reply]
就是网页端的与ChatGPT对话的内容日志

**Comment by dragonforce2010** [comment]
群都满员了，哪个大佬可以拉一下我379890082🙏

**Comment by hunduncn** [comment]
群都满了。

**Comment by ijry** [comment]
谁能拉我下598821125

**Comment by All8926** [comment]
全都满了，求拉一下，3181389202

**Comment by kong1019** [comment]
好鸽鸽拉下我,2830821747

**Comment by zhongjuntao** [comment]
群主满了 拉一下我 1103801592

**Comment by Jefflian12** [comment]
9群都满了，求拉893856512

**Comment by shandianke1** [comment]
群都满员了，哪个大佬可以拉一下我5180932🙏

**Comment by Astlvk** [comment]
你好，群满了，期待10群。

**Comment by anyonecanpalyai** [comment]
群都满员了，哪个大佬可以拉一下我372750239

**Comment by Yidadaa** [comment]
10 群：820958210

**Comment by abner-wong** [comment]
10群已满，求新群

**Comment by linglom1** [comment]
请问还会有新群吗

**Comment by devyujie** [comment]
求新群

**Comment by xlongshu** [comment]
求新群  或者TG群 QQ这样扛不住啊

**Comment by Yidadaa** [comment]
QQ 群新增 2000 人群：617286615

**Comment by Yidadaa** [comment]
QQ 群最新 2000 人群：594121234

**Reply by jzcPandal** [reply]
群满了，求新群

**Comment by rifnjigmail** [comment]
ok

Yifei Zhang ***@***.***> 于2023年11月8日周三 18:21写道：

**Comment by Friedrich-hue** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by RobertCheng-956** [comment]
群满了，请问还有新群嘛？

**Reply by Jethuit** [reply]
蹲，有了踢我一下

**Comment by xosi** [comment]
群满了！没有新群要不大家加我这个群吧，是新建的，大家一起讨论 63291913

**Comment by xiaoheilflqs** [comment]
能拉我下吗 170255841

**Reply by dxzxomy** [reply]
群满了，能拉下吗，903248178

**Comment by comqx** [comment]
能拉我下吗 345186121

**Comment by FelixMotorcycle** [comment]
咋啦啊亲，我小白不懂啊

comqx ***@***.***>于2024年1月17日 周三09:42写道：

**Comment by ifp1090** [comment]
群已经满了无法加入

**Comment by Friedrich-hue** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by oicoderr** [comment]
满了，麻烦拉下 849656290

**Comment by Madilynalisa** [comment]
满了，麻烦大家拉一下 1227360183

**Comment by Friedrich-hue** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by ersaijun** [comment]
我自己搭好了，不想运营了，有要要的吗 网址是 free.leapgpt.top 和 chat4.leapgpt.top

**Comment by Friedrich-hue** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by echo368** [comment]
有新群可以拉我一下吗 1783633063

**Comment by wangxiaoqian2042** [comment]
请求建立 discord 频道

**Comment by CZILYYZ** [comment]
满了，求拉302858879

**Comment by ilyc157** [comment]
596475044 麻烦拉下

**Comment by Friedrich-hue** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by bygsn** [comment]
> 为了提高交流效率，我们设立了官方 QQ 群和 QQ 频道，如果你在使用或者搭建过程中遇到了任何问题，请先第一时间加群或者频道咨询解决，除非是可以稳定复现的 Bug 或者较为有创意的功能建议，否则请不要随意往 Issue 区发送低质无意义帖子。
> 
> 点击下方蓝色字体加入对应群聊，二维码会定时更新。
> 
> [QQ 频道](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/16968934/49b9ada0-c9b3-4891-8a79-59340f947589) ｜ [QQ 群](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/16968934/31f3ff9f-d62e-432b-9f84-5e2fc6b49103)
> 
> 如果二维码过期，请尝试搜索 QQ 频道：`chatgptn4web`，QQ 群：`594121234` 。

请求建立微信群 qq基本不用

**Comment by Friedrich-hue** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by flyyangX** [comment]
群满了 求拉381098932

**Comment by Friedrich-hue** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by cuianning** [comment]
QQ群已满，请求建立微信群，QQ 414419752

**Comment by Friedrich-hue** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
